editor/editor_run_native.cpp

```diff
@@ -61,6 +61,7 @@ void EditorRunNative::_notification(int p_what) {
 						}
 					}
 				}
+				remote_debug->set_disabled(remote_debug->get_popup()->get_item_count() == 0);
```

Comment:
I would change the tooltip text when disabled to make it clear.
```suggestion
				if (remote_debug->get_popup()->get_item_count() == 0) {
					remote_debug->set_disabled(true);
					remote_debug->set_tooltip_text(TTR("No Remote Debug export presets configured."));
				} else {
					remote_debug->set_disable(false);
					remote_debug->set_tooltip_text(TTR("Remote Debug"));
				}
```
BTW, `remote_debug->get_popup()` looks like a good candidate for a local variable given the code above.

Reply:
Done.